### PR TITLE
changes to setup

### PR DESCRIPTION
**Pull request by mgrosen**
- updates go install 
- comments out bionic references since it messes up python bcc installation
- ran chmod +x on the files to make them executable

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -10,22 +10,23 @@ sudo apt-get install -y bpfcc-tools linux-headers-$(uname -r)
...
-sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4052245BD4284CDD
-echo "deb https://repo.iovisor.org/apt/bionic bionic main" | sudo tee /etc/apt/sources.list.d/iovisor.list
-sudo apt-get update
-sudo apt-get install -y bcc-tools libbcc-examples linux-headers-$(uname -r)
+#sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4052245BD4284CDD
+#echo "deb https://repo.iovisor.org/apt/bionic bionic main" | sudo tee /etc/apt/sources.list.d/iovisor.list
+#sudo apt-get update
+#sudo apt-get install -y bcc-tools libbcc-examples linux-headers-$(uname -r)
 
 sudo /usr/local/go/bin/go get -u ./...
```
Comment (TommasoPieroncini) [inline setup.sh]: Don't we have to edit the go mod file sometime before this? 

> We ran this go list -m github.com/iovisor/gobpf@master and grabbed version v0.2.1-0.20221005153822-16120a1bf4d4

**Review by TommasoPieroncini — APPROVED** [review]

**Comment by TommasoPieroncini** [comment]
Says setup_python_ebpf.sh is an empty file, but when I pulled your branch I saw the code 😕

File: setup.sh (hunk 1 above)
Comment (TommasoPieroncini) [inline setup.sh]: I get this error when running this command after going through the other steps:

> go: go.mod file not found in current directory or any parent directory.

File: setup.sh [hunk 2]
```diff
@@ -10,22 +10,23 @@ sudo apt-get install -y bpfcc-tools linux-headers-$(uname -r)
...
 export CGO_ENABLED=1
 sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
   libllvm12 llvm-12-dev libclang-12-dev python zlib1g-dev libelf-dev libfl-dev python3-setuptools \
   liblzma-dev arping netperf iperf
 
-sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4052245BD4284CDD
-echo "deb https://repo.iovisor.org/apt/bionic bionic main" | sudo tee /etc/apt/sources.list.d/iovisor.list
-sudo apt-get update
-sudo apt-get install -y bcc-tools libbcc-examples linux-headers-$(uname -r)
+#sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4052245BD4284CDD
```
Comment (TommasoPieroncini) [inline setup.sh]: FYI that without these commands, my setup failed. Running these commands fixed it. Might just be that we need libbcc, but anyway I suggest leaving these if possible

https://github.slack.com/archives/C05GZ7UL7NK/p1689775348572379

**Review by TommasoPieroncini — CHANGES_REQUESTED** [review]
feel free to ignore, but i left some comments where these changes to setup are breaking the python program at least

File: setup.sh (hunk 1 above)
Comment (mgrosen) [reply setup.sh]: its already updated in this repo!

Comment (mgrosen) [reply setup.sh]: weird, worked for me when I ran from the highest level folder of the repo

**Pull request merged**